website/docs/d/vra_region.html.markdown

```diff
@@ -0,0 +1,27 @@
+---layout: "vra"
+page_title: "VMware vRealize Automation: vra_region_enumeration"
+description: |-
+  Provides a data lookup for vra_region_enumeration.
+---
+
+# Data Source: vra_region_enumeration
+## Example Usages
+
+This is an example of how to read a region enumeration data source.
+
+DeprecationMessage: 'region_enumeration' is deprecated. Use 'region_enumeration_vsphere' instead.
+
+## Argument Reference
+* `dcId` - Identifier of a data collector vm deployed in the on premise infrastructure. Example: d5316b00-f3b8-4895-9e9a-c4b98649c2ca
+
+## Attribute Reference
+* `accept_self_signed_cert` - Accept self signed certificate when connecting to vSphere. Example: false
+
+* `hostname` - Host name for the cloud account endpoint. Example: dc1-lnd.mycompany.com
+
+* `password` - Password for the user used to authenticate with the cloud Account
+
+* `username` - Username to authenticate with the cloud account
+
+* `regions` - A set of datacenter managed object reference identifiers to enable provisioning on. Example: Datacenter:datacenter-2 
+
```

Comment:
Add example usage and also update the arguments and attributes as per `vra_region` data source.